src/componentes/Home/Home.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+
+const Home = () => (
+<div className='home'>
+        <Header></Header>
+        <Footer></Footer>
+</div>
```

Comment:
remover espacios adicionales
```suggestion
<div className='home'>
  <Header></Header>
  <Footer></Footer>
</div>
```